### ISSUE TITLE
Creating a {descriptor, class} object dataset 

### ISSUE THREAD

**Issue by nikhilbv** [fabvio/Cascade-LD#8]
Hi Fabvio,

Great work. I would love to know when the training code will be released. Meanwhile, I'm trying to train only the classifier CNN and for that, I'm trying to create a {descriptor, class} dataset as mentioned in the paper. In my approach, I'm directly drawing the lanes on an image(binary) from the tusimple dataset lane annotations, resizing the original and binary image to 360*640, converting it to torch tensors, and passing it to extract_descriptor function.

Expected Output - 
For a four-lane image, four descriptors should be extracted

Actual output - 
No descriptor is extracted

I am wondering if my approach is correct and if it is correct, then why is the extract_descriptor function is not working as expected.

**Comment by fabvio** [comment]
Hi. I guess that the function is not working because you are passing a binary image! Check the jupyter notebook and you'll see that you need an image with instance labels projected on in order to extract the descriptors :)

**Comment by nikhilbv** [comment]
Hi! Thanks for the reply. Does the width of the lane matters in the image with instance labels projected? I have made an instance image using tusimple dataset annotations. It looks like this.
![0](https://user-images.githubusercontent.com/40387848/82298558-ece70880-99d1-11ea-921e-a92e68e3868d.png).
I've used the width of 5px for drawing polyline but still not able to extract descriptors

**Comment by fabvio** [comment]
This is the actual image you are passing to the function? Because you shouldn't pass a color image, you should pass a greyscale one with contiguous lane ids, e.g. 1,2,3,4 instead of (0,255,0),(255,0,0) etc.

**Comment by fabvio** [comment]
Closed for inactivity. Feel free to reopen if there is the need.

**Comment by jiguanglu** [comment]
> trying to train only the classifier CNN
Hello, can you share the code for the training classifier CNN?  I will be very grateful to you.

**Comment by jiguanglu** [comment]
> This is the actual image you are passing to the function? Because you shouldn't pass a color image, you should pass a greyscale one with contiguous lane ids, e.g. 1,2,3,4 instead of (0,255,0),(255,0,0) etc.

Hello, can you share the code for the training classifier CNN? I will be very grateful to you.
